### PR TITLE
#1312 Removed models from accounts app

### LINKED ISSUE #1312: Remove models from Accounts app
**Expected Behaviour** : Remove the `Affiliation` and `UserAffiliation` model from the `Accounts` app.
**Deliverables:**
- [ ] Remove the `Affiliation` and `UserAffiliation` model from accounts app.
- [ ] Create and name the migration.

### PR DESCRIPTION
**Pull request by HargovindArora**
"Fix #1312" 
- Accounts models.py updated
- New Migration created and named

### CODE REVIEW

**Comment by HargovindArora** [comment]
Please, someone, review it! :)

**Comment by RishabhJain2018** [comment]
@HargovindArora I will do it by tonight.

**Comment by HargovindArora** [comment]
@RishabhJain2018 Sure. :smile:

File: apps/accounts/migrations/0003_affiliation_remove.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
```
Comment (RishabhJain2018) [inline apps/accounts/migrations/0003_affiliation_remove.py]: @HargovindArora Can you please rename the migration as `remove_affiliation_model.py`

Comment (HargovindArora) [reply apps/accounts/migrations/0003_affiliation_remove.py]: @RishabhJain2018 Sure :smile:

**Comment by RishabhJain2018** [comment]
@deshraj Please review and merge it.

**Pull request merged**